config/crd/skiperator.kartverket.no_applications.yaml

```
@@ -553,6 +553,83 @@ spec:
                     maximum: 3600
                     minimum: 1
                     type: integer
+                  authentication:
+                    description: Authentication specifies how incoming JWT's should
```

Nitpick: Use plural and not possessive form:
```suggestion
                    description: Authentication specifies how incoming JWTs should
```